src/__tests__/github.test.ts

```diff
@@ -151,5 +151,44 @@ test("getIssueContent() requests GitHub's API, when issueNumber is set", async (
     title: 'test-title',
     body: 'test-body',
     labels: ['needs-triage'],
+    isValidIssueType: true,
   });
 });
+
+test('VerifyIssueType returns true when target is both', async () => {
+  process.env.INPUT_TARGET = 'both';
+
+  const githubApi = new GithubApi('GITHUB_TOKEN');
+
+  const data = {
+    pull_request: ['pullrequest'],
+  };
```

Comment:
```suggestion
test('VerifyIssueType returns true when target is both', async () => {
  process.env.INPUT_TARGET = 'both';
  const data = {
    pull_request: ['pullrequest'],
  };

  const githubApi = new GithubApi('GITHUB_TOKEN');
```
AAA & spacing ⛏️

Reply:
Dang I remembered to do that for the second two tests I put! Almost had it 😤